### ISSUE TITLE
🐛 Bug Report: Collection attribute with name 'order' causes exception on listDocuments()

### ISSUE THREAD

**Issue by ddenev** [appwrite/appwrite#3250]
### 👟 Reproduction steps

I have a collection that contains an integer attribute named 'order'.
When I call `listDocuments()` like this (Web API):
```js
appwrite.database.listDocuments('budget-categories', [], 100, 0, '', 'after', ['order'], ['ASC'])
```
I get the following exception:
```
Access to XMLHttpRequest at 'http://192.168.88.11/v1/database/collections/budget-categories/documents?limit=100&offset=0&cursor=&cursorDirection=after&orderAttributes%5B0%5D=order&orderTypes%5B0%5D=ASC' from origin 'http://localhost:9000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
AppwriteException: Network request failed
    at Appwrite.<anonymous> (http://localhost:9000/node_modules/.q-cache/vite/spa/deps/appwrite.js?v=af4ab88e:1985:15)
    at Generator.throw (<anonymous>)
    at rejected (http://localhost:9000/node_modules/.q-cache/vite/spa/deps/appwrite.js?v=af4ab88e:495:32)
GET http://192.168.88.11/v1/database/collections/budget-categories/documents?limit=100&offset=0&cursor=&cursorDirection=after&orderAttributes%5B0%5D=order&orderTypes%5B0%5D=ASC net::ERR_FAILED 500
```
This only happens with the 'order' attribute. If I change to any other attribute from the collection, it works.

### 👍 Expected behavior

No exception thrown when ordering by the 'order' attribute

### 👎 Actual Behavior

See reproduction above

### 🎲 Appwrite version

Version 0.14.x

### 💻 Operating system

Windows

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
For the core team, I tested this on 0.13.4 and got the follow in my appwrite logs:

```
appwrite  | [Error] Type: PDOException
appwrite  | [Error] Message: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'order DESC, table_main._id ASC
appwrite  |             LIMIT 0, 12' at line 21
appwrite  | [Error] File: @swoole-src/library/core/Database/PDOStatementProxy.php
appwrite  | [Error] Line: 64
```

**Comment by jkklapp** [comment]
Seems like there's a collision between the `order` parameter name in the collection and the `order` SQL clause

**Comment by stnguyen90** [comment]
[utopia-php/database@0.18.9](https://github.com/utopia-php/database/releases/tag/0.18.9) doesn't have the fix. Waiting for the next version of utopia-php/database to be released and then Appwrite to upgrade.

**Comment by ddenev** [comment]
I see there is 1.0.0 coming. Can this fix be added to the 1.0.0 milestone if 0.16.0 is not going to happen?

**Comment by stnguyen90** [comment]
@ddenev, yes this will be in 1.0.0
